pyproject.toml

```
@@ -1,7 +1,104 @@
 [build-system]
 # this version is required to support reading of version in setup.cfg
-requires = ["setuptools>=46.4.0"]
-build-backend = "setuptools.build_meta"
+requires = ["flit_core >= 3.4,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "aiidalab"
+dynamic = ['version']
+authors = [{name = 'The AiiDAlab team', email = 'aiidalab@materialcloud.org'}]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: AiiDA",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: Implementation :: CPython",
+]
+dependencies = [
+    "cachetools~=4.1",
+    "click>=7.0,<9",
+    "click-spinner~=0.1",
+    "dulwich~=0.20",
+    "packaging>=20.1",
+    "pip",
+    "requests~=2.27",
+    "requests-cache~=1.0",
+    "setuptools",
+    "tabulate~=0.8",
+    "toml~=0.10",
+    "traitlets~=5.0",
+    "watchdog~=4.0",
+]
+description = "Implements core functions of AiiDAlab"
+license = {file = 'LICENSE'}
+readme = 'README.md'
+requires-python = '>=3.9'
+
+[project.script]
```

I tested by running `uv build` and got

```console
$ uv build
Building source distribution...
Unexpected names under [project]: script
Unexpected names under [project]: script
Building wheel from source distribution...
Unexpected names under [project]: script
Unexpected names under [project]: script
Successfully built dist/aiidalab-24.9.0.tar.gz
Successfully built dist/aiidalab-24.9.0-py3-none-any.whl
```

There's a typo here:

```suggestion
[project.scripts]
```

I am surprised that the tests are passing, since the `aiidalab` cli command is not available due to this mistake. We'll need to expand the test to catch this.